Repository/Repository/Configurations/ContactEndpoints.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ContactEndpoints.swift
+//  Repository
+//
+//  Created by Hao Tran Thien on 06/01/2022.
+//
+
+import Foundation
+
+enum ContactEndpoints {
+    case fetchContacts(size: Int)
+}
+
+extension ContactEndpoints {
+    var apiEndpointUrl: String {
```

Comment:
We shouldn't to use singleton here, it will make dependency in Api class ? We should create BaseApi and will pass this value through DI (Repository class).